clients/admin-ui/src/features/system/dictionary-form/DictSuggestionTextInput.tsx

```diff
@@ -46,16 +47,19 @@ const useDictSuggestion = (fieldName: string, dictField: string) => {
     ) {
       if (field.value !== dictEntry[dictField as keyof DictEntry]) {
         setValue(dictEntry[dictField as keyof DictEntry]);
+
+        // This blur is a workaround some forik issues.
+        // the setTimeout is required to get around a
+        // timing issue with the ref not being ready yet.
+        setTimeout(() => {
```

Comment:
oof nice job finding a workaround for this, how annoying! I poked at it for a bit but couldn't find a better solution, so let's roll with it for now!